content/v2.0/reference/cli/influx/telegrafs/rm.md

```diff
@@ -0,0 +1,37 @@
+---
+title: influx telegrafs rm
+description: >
+ placeholder
+menu:
+  v2_0_ref:
+    name: influx telegrafs rm
+    parent: influx telegrafs
+weight: 201
+---
+
+The `influx telegrafs rm` command removes Telegraf configurations from InfluxDB.
+
+## Usage
+```sh
+influx telegrafs rm [flags]
+```
+
+#### Aliases
+`rm`, `remove`
+
+## Flags
+| Flag |                  | Description                         | Input type  | {{< cli/mapped >}}     |
+|:---- |:---              |:-----------                         |:----------: |:------------------     |
+| `-h` | `--help`         | Help for the `rm` command           |             |                        |
+|      | `--hide-headers` | Hide the table headers              |             | `$INFLUX_HIDE_HEADERS` |
+| `-i` | `--id`           | Telegraf configuration ID to remove | strings     |                        |
+|      | `--json`         | Output data as json                 |             | `$INFLUX_OUTPUT_JSON`  |
```

Comment:
Curious about when you'd want to output json if you're trying to remove configuration?

Reply:
This is a standard implementation across the entire CLI. If a user is scripting the CLI, JSON output is much easier to parse than the standard output.